frontend/app/routes/_gcweb-app.personal-information.preferred-language.confirm.tsx

```diff
@@ -54,14 +50,12 @@ export default function PreferredLanguageConfirm() {
       </h1>
       <p>{t('personal-information:preferred-language.confirm.confirm-message')}</p>
       <Form method="post">
+        <dl>
+          <dt>{t('personal-information:preferred-language.edit.language')}</dt>
+          <dd>{loaderData.preferredLanguage === 'en' ? t('personal-information:preferred-language.edit.nameEn') : t('personal-information:preferred-language.edit.nameFr')}</dd>
+        </dl>
         <div className="form-group">
-          <dl>
-            <dt>{t('personal-information:preferred-language.index.language')}</dt>
-            <dd>{loaderData.preferredLanguage === 'en' ? t('personal-information:preferred-language.index.nameEn') : t('personal-information:preferred-language.index.nameFr')}</dd>
-          </dl>
-        </div>
-        <div className="flex flex-wrap gap-3">
-          <button className="btn btn-primary btn-lg">{t('personal-information:preferred-language.confirm.button.confirm')}</button>
+          <button className="btn btn-primary btn-lg mrgn-rght-sm">{t('personal-information:preferred-language.confirm.button.confirm')}</button>
```

Comment:
Based on https://github.com/DTS-STN/canadian-dental-care-plan/pull/194, the `<div className="flex flex-wrap gap-3">` should be kept.